test/hotspot/jtreg/TEST.groups

```diff
@@ -399,6 +399,17 @@ hotspot_cds_verify_shared_spaces = \
   runtime/cds/appcds/jvmti/dumpingWithAgent/DumpingWithJavaAgent.java \
   runtime/cds/appcds/sharedStrings/SharedStringsBasic.java
 
+hotspot_cds_epsilongc = \
```

Comment:
I don't know why we don't comment groups in this file but please add a comment here that these are low-memory using tests that won't fail (with a reasonable heap size) when run with a fake GC.

Reply:
I added a comment.